Updating dependencies

cabbiepete opened the pull request:
This is still useful as reference so I updated the dependancies and node versions to test against so its a better reference.

Commit:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:send:20151103
- https://snyk.io/vuln/npm:send:20140912
- https://snyk.io/vuln/npm:qs:20140806
- https://snyk.io/vuln/npm:qs:20140806-1
- https://snyk.io/vuln/npm:cookie-signature:20160804
- https://snyk.io/vuln/npm:express:20140912
- https://snyk.io/vuln/npm:connect:20130701
- https://snyk.io/vuln/npm:ms:20151024
- https://snyk.io/vuln/npm:mongoose:20160116

Latest report for cabbiepete/backbone-testing:
https://snyk.io/test/github/cabbiepete/backbone-testing

Commit:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:negotiator:20160616
- https://snyk.io/vuln/npm:send:20151103
- https://snyk.io/vuln/npm:serve-static:20150120
- https://snyk.io/vuln/npm:serve-static:20150113
- https://snyk.io/vuln/npm:send:20140912
- https://snyk.io/vuln/npm:qs:20140806-1
- https://snyk.io/vuln/npm:qs:20140806
- https://snyk.io/vuln/npm:express:20140912
- https://snyk.io/vuln/npm:cookie-signature:20160804
- https://snyk.io/vuln/npm:mongoose:20160116

Latest report for cabbiepete/backbone-testing:
https://snyk.io/test/github/cabbiepete/backbone-testing

cabbiepete committed:
Merge pull request #1 from cabbiepete/snyk-fix-d4178f9f

[Snyk] Fix for 12 vulnerable dependency paths

cabbiepete committed:
JSHint fix

cabbiepete committed:
Updating dependancies

cabbiepete committed:
Add newer nodes to travis list.

ryan-roemer commented:
@cabbiepete -- Thanks for chipping in! Can you apply this CI update diff to your PR and see if that fixes what's ailing Travis / Sauce Labs?

https://github.com/ryan-roemer/backbone-testing/compare/chore-updateTravis.diff

cabbiepete committed:
see if we can get travis working

cabbiepete commented:
@ryan-roemer Travis complaining with "No binary for Chrome browser on your platform" for sauce runner tests. Any ideas?

ryan-roemer commented on .travis.yml:
```diff
@@ -9,15 +12,20 @@ before_script:
-    - BB_TESTING_CHECK=1
-    - BB_TESTING_KARMA=ci
-    - BB_TESTING_KARMA=sauce
-    - BB_TESTING_KARMA=bs
```
L17-L19 need to be reverted. Only L20 (`BB_TESTING_KARMA=bs`) is replaced with new L20-L22.

ryan-roemer reviewed (changes requested):
Have some travis fixes that may help...

Commit:
fix: package.json to reduce vulnerabilities

The following vulnerabilities are fixed with an upgrade:
- https://snyk.io/vuln/npm:qs:20170213

Latest report for cabbiepete/backbone-testing:
https://snyk.io/test/github/cabbiepete/backbone-testing

cabbiepete committed:
Merge pull request #2 from cabbiepete/snyk-fix-73bce91f

[Snyk Update] New fixes for 1 vulnerable dependency path
